deepomatic/cli/input_data.py

```diff
@@ -129,6 +137,68 @@ def is_infinite(self):
         return False
 
 
+class StudioInputData(InputData):
+    @classmethod
+    def is_valid(cls, descriptor):
+        _, ext = os.path.splitext(descriptor)
+        return os.path.exists(descriptor) and ext.lower() in SUPPORTED_STUDIO_INPUT_FORMAT
+
+    def __init__(self, descriptor, **kwargs):
+        super(StudioInputData, self).__init__(descriptor, **kwargs)
+        self._frames = []
+        self._studio_file_dir = os.path.dirname(self._descriptor)
+        self._name = 'studio_%s_%s' % ('%05d', self._reco)
+        self._iterator = None
+    
+    def __iter__(self):
+        self._frames = []
+        with open(self._descriptor) as f:
+            for line_i, line in enumerate(f.readlines()):
```

Comment:
Avoid loading the file content in memory

```suggestion
            for line_i, line in enumerate(f):
```